kubric/renderer/blender.py

```diff
@@ -737,3 +747,6 @@ def register_object3d_setters(obj, blender_obj):
 
   obj.observe(AttributeSetter(blender_obj, "rotation_quaternion"), "quaternion")
   obj.observe(KeyframeSetter(blender_obj, "rotation_quaternion"), "quaternion", type="keyframe")
+
+  # --- Maps Object3D.cast_shadows to bpy.context.object.cycles_visibility.shadow
+  obj.observe(VisibleShadowsSetter(blender_obj), "cast_shadows")
```

Comment:
I think this should work without the need for a dedicated `VisibleShadowsSetter`:
``` python
obj.observe(AttributeSetter(blender_obj.cycles_visibility, "shadow"), "cast_shadows")
```